features/provider/admin/account/authentication_providers/new.feature

```diff
@@ -6,9 +6,19 @@ Feature: Account Settings > Users > SSO Integrations > New
 
   Scenario: Navigation
     Given they go to the users sso integrations page
-    When they follow "Create a new SSO integration"
+    When they follow "Add a SSO integration"
     Then the current page is the new sso integration page
 
+  Scenario: Navigation when there is an integration
+    Given a red hat single sign-on integration
+    And they go to the users sso integrations page
+    Then there should be a link to "Create a new SSO integration"
+
+  Scenario: Empty state
```

Comment:
I've added a step `Then the should not see "Add a SSO integration"`, so it's clear, since the `Scenario: Navigation when there is an integration` was already meant to test that.